testutil/session/session.go

```diff
@@ -0,0 +1,49 @@
+package session
+
+import (
+	"github.com/pokt-network/poktroll/x/session/keeper"
+	"github.com/pokt-network/poktroll/x/shared"
+	"github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// GetSessionIdWithDefaultParams returns the string and bytes representation of the
+// sessionId for the session containing blockHeight, given the default shared on-chain
+// parameters, application public key, service ID, and block hash.
+func GetSessionIdWithDefaultParams(
+	appPubKey,
+	serviceId string,
+	blockHashBz []byte,
+	blockHeight int64,
+) (sessionId string, sessionIdBz []byte) {
+	sharedParams := types.DefaultParams()
+	return keeper.GetSessionId(&sharedParams, appPubKey, serviceId, blockHashBz, blockHeight)
+}
+
+// GetSessionStartHeightWithDefaultParams returns the block height at which the
+// session containing queryHeight starts, given the default shared on-chain
+// parameters.
+// Returns 0 if the block height is not a consensus produced block.
```

Comment:
See the comments I left in the other PR.

I don't think we should copy-pasta the details of the return and example for the `WithDefaultParams` functions. 

Instead - refer to the underlying function.